docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -94,12 +94,15 @@ error | When this style is not being followed, show a compiler error.
     - ["Null" Checking Preferences](#null_checking)
         - [Throw-expressions](#null_checking_throw_expressions)
         - [Conditional delegate calls](#null_checking_conditional_delegate_calls)
+    - [Code Block Preferences](#code_block)
+        - [Prefer braces](#prefer_braces)
 
 ## <a name="this_and_me">"This." and "Me." Qualification</a>
 ### <a name="this_and_me_fields">Fields (IDE0003/IDE0009)</a>
-|  Option Name | `dotnet_style_qualification_for_field` |
-| ------------- |:-------------:|
-| **Applicable Languages** | C# and Visual Basic
+| **Option Name** | **Applicable Languages** | **Visual Studio Default** | **Supported Version** |
+| ----------- | -------------------- | ----------------------| ----------------  |
+|  `dotnet_style_qualification_for_field` | C# and Visual Basic | false:none | Visual Studio 2017 RTW |

```

Comment:
Updates are worded like this: Visual Studio 2017 version 15.3. But I think for the table, abbreviating "version" to "v." as you have done in other changes is fine.